modules/setup-systemd-resolved/setup-systemd-resolved

```diff
@@ -67,14 +67,30 @@ function assert_not_empty {
   fi
 }
 
+function has_yum {
+  [ -n "$(command -v yum)" ]
```

Comment:
NIT: we follow the [Google Bash style guide](https://google.github.io/styleguide/shellguide.html), so would you mind updating to `[[ ]]` instead of `[ ]`?